packages/bot-skeleton/src/scratch/utils/index.js

```diff
@@ -14,17 +17,18 @@ export const oppositesToDropdownOptions = opposite_name => {
     });
 };
 
-export const cleanUpOnLoad = (blocks_to_clean, drop_event) => {
+export const cleanUpOnLoad = (blocks_to_clean, drop_event, preview_workspace) => {
+    const workspace = preview_workspace || Blockly.derivWorkspace;
```

Comment:
Better to pass in the workspace that needs to be cleaned up, rather than falling back to a global.